sandbox/glassdoor-scraping/glassdoor-scraper.py

```diff
@@ -13,8 +13,8 @@
 from GlassdoorCsvRow import GlassdoorCsvRow
 import boto3
 
-username = "####" # your email here
-password = "####" # your password here
+username = "###" # your email here
```

Comment:
this change was unnecessary.. I was using my creds and when I got rid of them, changed the number of these characters..